src/main/java/step1/calculator/model/Operation.java

```diff
@@ -0,0 +1,33 @@
+package step1.calculator.model;
+
+import java.util.Map;
+
+public enum Operation {
+
+    ADD {
+        @Override
+        public int calculate(int x, int y) {
+            return x + y;
+        }
+    }, SUBTRACT {
+        @Override
+        public int calculate(int x, int y) {
+            return x - y;
+        }
+    }, MULTIPLY {
+        @Override
+        public int calculate(int x, int y) {
+            return x * y;
+        }
+    }, DIVIDE {
+        @Override
+        public int calculate(int x, int y) {
+            return x / y;
+        }
+    };
+
+    public static final Map<String, Operation> OPERATIONS = Map.of("+", ADD, "-", SUBTRACT, "*", MULTIPLY, "/", DIVIDE);
+
+    public abstract int calculate(int x, int y);
```

Comment:
```suggestion
    private final BiFunction<Integer, Integer, Integer> operation;
```

자바의 기본 함수형 인터페이스를 활용해보는것도 좋을것 같아요 :)